Fix EventState Exclusive parsing interface{} to *bool error

VicNgu opened the pull request:
**What this PR does / why we need it**:
Currently, when the exclusive flag is provided, the code will panic due to interface{} unable to be parsed into a (*bool). This change defaults the exclusive flag to true and sets the value if it is provided as a bool.

ricardozanini reviewed (commented):
Many thanks! Can you please [add a unit test](https://github.com/serverlessworkflow/sdk-go/blob/main/parser/parser_test.go#L41) to verify this change?  

Ideally, with an example that would fire a nil exception.

VicNgu commented:
> Many thanks! Can you please [add a unit test](https://github.com/serverlessworkflow/sdk-go/blob/main/parser/parser_test.go#L41) to verify this change?
> 
> Ideally, with an example that would fire a nil exception.

Added two test cases and updated the existing one.
The existing test case did not have the exclusive flag set, so it should default to true.
Other two test cases explicitly sets exclusive to true and false.
